th/2017/winter.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="fr">
+	<head>
+		<title>ฤดูหนาว 2017 | ไทย | The Fansub Database</title>
```

Comment:
Can you rename ALL winter 2017 to **winter 2016** please?
For me winter 2016/2017 on livechart (https://www.livechart.me/winter-2017/tv) is winter 2016.

I don't know if it's a good idea, but I've began to do the website like that.